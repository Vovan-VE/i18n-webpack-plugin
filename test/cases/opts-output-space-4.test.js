import fs from 'fs';
import processFile from '../cases.setup';

describe('opts-output-space-4', () => {
  const extracted = [];
  const outputSpace = 4;

  beforeAll(() => processFile('opts-output-space-4.code.js', { outputSpace })
    .then(({ files }) => {
      Object.keys(files).forEach((name) => {
        extracted.push({
          name,
          content: fs.readFileSync(files[name], 'utf8'),
        });
      });
    }));

  it('should return extracted keys', () => {
    expect(extracted).toMatchSnapshot();
  });
});
