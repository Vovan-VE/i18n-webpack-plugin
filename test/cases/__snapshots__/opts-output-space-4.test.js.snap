// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`opts-output-space-4 should return extracted keys 1`] = `
"{
    \\"category1\\": {
        \\"Message 1\\": \\"\\",
        \\"Message 2\\": \\"\\"
    },
    \\"category2\\": {
        \\"Message 3\\": \\"\\"
    }
}"
`;
